src/pynteny/cli.py

```diff
@@ -122,7 +122,7 @@ def download(self):
     def app(self):
         """Run pynteny app through Streamlit"""
         parser = SubcommandParser.app()
-        args = parser.parse_args(self._subcommand_args)
+        parser.parse_args(self._subcommand_args)
```

Comment:
parser required here for `pynteny app --help` to work (consistency across subcommand)